csaf_2.1/prose/edit/src/safety-security-and-data-protection.md

```diff
@@ -31,4 +31,16 @@ Moreover, it SHALL be treated as unsafe (user) input.
   > Additional, supporting mitigation measures like retrieving only CSAF documents from trusted sources and check their integrity and
   > signature before parsing the document SHOULD be in place to reduce the risk further.
 
+The distribution requirements of CSAF data allow to specify domains as the value of the HTTP header `Access-Control-Allow-Origin`.
+While a wildcard (`*`) as header value usually prevents implementing browsers from sending credentials during the CORS request,
+the restriction to specified domains often enables sending credentials.
+As such configuration is more complex, there is a higher risk of misconfiguration.
```

Comment:
Better: `Allowing several specific domains means using dynamics on the server, which can widen the attack surface slightly by more active code and configuration.`

Reply:
Addressed with [62144c5](https://github.com/oasis-tcs/csaf/pull/903/commits/62144c5c728a2f1be4dc6d108cecfe22640f7e8f)